### DISCUSSION TITLE
an error in seed coordinates

### DISCUSSION THREAD

**Discussion by neperchh** [neperfepx/neper#686] [Q&A] [answered]
dear professor
 I want to generate a model using my seed coordinates from EBSD dates. However, when I  input 4 coordinate points randomly, the output is only one rectangle.When I used my EBSD dates,I cannot obtain .tess file.The following is my code and random seed coordinate and results. Can you help me? Thank you very much.
code : neper -T -n 4 -morphooptiini "coo:file(coo1),weight:0" -morpho voronoi -dim 2
seed coordinate：
1 10
2 5
3 7
4 6
![屏幕截图 2023-04-19 150534](https://user-images.githubusercontent.com/127730078/232994292-ad5a75c0-41cd-4e33-bd3e-1f4f74b2dd59.png)

**Comment by rquey** [comment, marked as answer]
You are working in the default, unit square domain.  Use `-domain` to specify the actual shape/size.
